pkg/3scale/amp/component/apicast.go

```diff
@@ -16,6 +18,9 @@ import (
 const (
 	ApicastStagingName    = "apicast-staging"
 	ApicastProductionName = "apicast-production"
+
+	CustomPoliciesMountBasePath    = "/opt/app-root/src/policies"
+	CustomPoliciesAnnotationPrefix = "apps.3scale.net/apicast-policy-volume"
```

Comment:
The name of the constant is kind of confusing. I understand why it's been named with "prefix", but prefix is the part before the `/` https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/.
So this is really a partial annotation key and not a prefix

Reply:
Fair enough. I have updated. It is the prefix of the key of the annotation, hence: `CustomPoliciesAnnotationKeyPrefix`